bingo/bingo-elastic/python/bingo_elastic/elastic.py

```diff
@@ -133,7 +134,9 @@ async def a_create_index(
     index_name: str, el_client: "AsyncElasticsearch"
 ) -> None:
     try:
-        await el_client.indices.create(index=index_name, body=index_body)
+        await el_client.indices.create(
+            index=index_name, body=index_body, ignore=400
```

Comment:
What does this `ignore=400` mean?

Reply:
@mkviatkovskii it’s ignoring 400 http exceptions from Elasticsearch. Not sure why we would need to do this

Reply:
I added this parameter in order to get rid of long and ugly error logs "index already created". It's not neccessary now, so I removed it too, thanks!